lib/org/openpsa/user/handler/person/account.php

```diff
@@ -78,6 +78,15 @@ private function load_controller() : controller
             ->get_controller();
     }
 
+    public function _handler_welcome_email(string $guid)
+    {
+        $this->person = new midcom_db_person($guid);
```

Comment:
This function is pretty standalone and doesn't call anything else, so we might as well use `$person = new midcom_db_person($guid);` instead of the class member

Reply:
changed